docs/standard/numerics.md

```diff
@@ -14,63 +14,68 @@ ms.assetid: dfebc18e-acde-4510-9fa7-9a0f4aa3bd11
 ---
 # Numerics in .NET
 
-.NET provides a range of numeric integer and floating-point primitives, as well as <xref:System.Numerics.BigInteger?displayProperty=nameWithType>, which is an integral type with no theoretical upper or lower bound, <xref:System.Numerics.Complex?displayProperty=nameWithType>, which represents complex numbers, and a set of SIMD-enabled types in the <xref:System.Numerics> namespace.
-  
+.NET provides a range of numeric integer and floating-point primitives, as well as:
+
+- <xref:System.Numerics.BigInteger?displayProperty=nameWithType>, which is an integral type with no theoretical upper or lower bound.
+- <xref:System.Numerics.Complex?displayProperty=nameWithType>, which represents complex numbers.
+- A set of SIMD-enabled types in the <xref:System.Numerics> namespace.
+
 ## Integer types
 
 .NET supports both signed and unsigned 8-, 16-, 32-, and 64-bit integer types, which are listed in the following table:
-  
-|Type|Signed/Unsigned|Size (in bytes)|Minimum value|Maximum value|  
-|----------|----------------------|--------------------|-------------------|-------------------|  
-|<xref:System.Byte?displayProperty=nameWithType>|Unsigned|1|0|255|  
-|<xref:System.Int16?displayProperty=nameWithType>|Signed|2|-32,768|32,767|  
-|<xref:System.Int32?displayProperty=nameWithType>|Signed|4|-2,147,483,648|2,147,483,647|  
-|<xref:System.Int64?displayProperty=nameWithType>|Signed|8|-9,223,372,036,854,775,808|9,223,372,036,854,775,807|  
-|<xref:System.SByte?displayProperty=nameWithType>|Signed|1|-128|127|  
-|<xref:System.UInt16?displayProperty=nameWithType>|Unsigned|2|0|65,535|  
-|<xref:System.UInt32?displayProperty=nameWithType>|Unsigned|4|0|4,294,967,295|  
-|<xref:System.UInt64?displayProperty=nameWithType>|Unsigned|8|0|18,446,744,073,709,551,615|  
-  
+
+|Type|Signed/Unsigned|Size (in bytes)|Minimum value|Maximum value|
+|----------|----------------------|--------------------|-------------------|-------------------|
+|<xref:System.Byte?displayProperty=nameWithType>|Unsigned|1|0|255|
+|<xref:System.Int16?displayProperty=nameWithType>|Signed|2|-32,768|32,767|
+|<xref:System.Int32?displayProperty=nameWithType>|Signed|4|-2,147,483,648|2,147,483,647|
+|<xref:System.Int64?displayProperty=nameWithType>|Signed|8|-9,223,372,036,854,775,808|9,223,372,036,854,775,807|
+|<xref:System.SByte?displayProperty=nameWithType>|Signed|1|-128|127|
+|<xref:System.UInt16?displayProperty=nameWithType>|Unsigned|2|0|65,535|
+|<xref:System.UInt32?displayProperty=nameWithType>|Unsigned|4|0|4,294,967,295|
+|<xref:System.UInt64?displayProperty=nameWithType>|Unsigned|8|0|18,446,744,073,709,551,615|
```

Comment:
Not related to this PR, but should `IntPtr` (`nint`) and `UIntPtr` (`nuint`) be listed here as well?

They are fundamental integer types, but have different ranges when running on 32-bit vs 64-bit